src/datasources/targeted-messaging/targeted-messaging.datasource.ts

```diff
@@ -60,6 +69,13 @@ export class TargetedMessagingDatasource
       name: outreach.name,
       startDate: new Date(outreach.start_date),
       endDate: new Date(outreach.end_date),
+      sourceId: outreach.source_id,
+      type: outreach.type,
+      teamName: outreach.team_name,
+      sourceFile: outreach.source_file,
+      sourceFileProcessedDate: outreach.source_file_processed_date
+        ? new Date(outreach.source_file_processed_date)
```

Comment:
Follow on from my type-related comment. Aren't these already `Date`s? (Would stand for `startDate`, `endDate`, `created_at` and `updated_at` as well.

Reply:
Absolutely! Answered in https://github.com/safe-global/safe-client-gateway/pull/2010#discussion_r1803861284